content/en/docs/tutorials/add-prebuilt-driver-config.md

```diff
@@ -0,0 +1,79 @@
+# Add prebuilt drivers for a kernel release
+
+## Requirements
+
+- Ensure the Linux distribution that distributes the Linux kernel version is supported by [driverkit](https://github.com/falcosecurity/driverkit/tree/master/pkg/driverbuilder/builder).
+- Ensure the driver is not already available for the specific Linux kernel version, by checking it at [download.falco.org/driver/site](https://download.falco.org/driver/site/index.html).
+
+## Generate the configuration
+
+You can generate and contribute configurations for your OS, as follows.
+
+Let's start by [forking](https://docs.github.com/en/get-started/quickstart/fork-a-repo) the [test-infra](https://github.com/falcosecurity/test-infra) repository.
+From the `driverkit` directory of the repository, run:
+
+```shell
+make generate -e TARGET_DISTRO=<TARGET_DISTRO> -e TARGET_KERNEL=<TARGET_KERNEL>
+```
+
+filling it with:
+* `TARGET_DISTRO`: the Linux distribution of the target host.
+* `TARGET_KERNEL`: the Linux kernel release version of the target host. You can fill `$TARGET_KERNEL` with the output of the command: `"$(uname -r)_$(uname -v | sed 's/#\([[:digit:]]\+\).*/\1/')"`, running it on a shell on the target host.
+
+for example:
+
+```shell
+make generate -e TARGET_DISTRO=arch -e TARGET_KERNEL=6.3.4-arch1-1
+```
+
+> Available values for `$TARGET_DISTRO` can be found [here](https://github.com/falcosecurity/driverkit/tree/master/pkg/driverbuilder/builder).
```

Comment:
Hey @FedeDP I'm wondering if we have a better documentation about the supported distributions in Driverkit.
Am I missing something here?

Reply:
We have this: https://github.com/falcosecurity/driverkit/blob/master/docs/driverkit.md (see the 
>   -t, --target string             the system to target the build for, one of [alinux,almalinux,amazonlinux,amazonlinux2,amazonlinux2022,amazonlinux2023,arch,bottlerocket,centos,debian,fedora,flatcar,minikube,ol,opensuse,photon,redhat,rocky,talos,ubuntu,vanilla]

)

Reply:
It gets updated with the `make docs` makefile target that should be run (we should add a ci check to run it IMHO)